labs/a4.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Lab 4 - Post Install
+layout: lab
+---
+
+# Generating SSH Keys
+Since passwords can be guessed, a secure SSH setup will usually use key-based authentication instead of (or in addition to) a passphrase. You may already have an ssh key, so run the following command to see if you have one:
+
+(note: We won’t give instructions here for Windows machines. To follow these instructions, SSH into `ssh.ocf.berkeley.edu` with your OCF account’s credentials and continue from there)
+
+```
+ls .ssh
+```
+
+If you see id_rsa or id_edsca listed, you already have an SSH key. If you don’t, run this command to generate one:
+
+
+```
+ssh-keygen -t rsa -b 4096
```

Comment:
Might be worth explaining why 4096 is chosen (I agree that it's the better choice) as the default is 2048.